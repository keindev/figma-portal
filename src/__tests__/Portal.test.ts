// @see https://github.com/facebook/jest/issues/9430
// eslint-disable-next-line node/no-extraneous-import
import { jest } from '@jest/globals';

jest.useFakeTimers();

describe('Portal', () => {
  it('fake-test', () => {
    expect(true).toBeTruthy();
  });
});
