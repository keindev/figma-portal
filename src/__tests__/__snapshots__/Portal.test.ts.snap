// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Portal Components extraction with File name 1`] = `
Array [
  Array [
    "https://example.com/images/logo",
  ],
  Array [
    "https://example.com/images/banner",
  ],
  Array [
    "media",
  ],
  Array [
    "media",
  ],
  Array [
    "media/logo.pdf",
    "",
  ],
  Array [
    "media/banner.pdf",
    "",
  ],
  Array [
    "https://example.com/images/logo",
  ],
  Array [
    "https://example.com/images/banner",
  ],
  Array [
    "media",
  ],
  Array [
    "media",
  ],
  Array [
    "media/logo.pdf",
    "",
  ],
  Array [
    "media/banner.pdf",
    "",
  ],
]
`;

exports[`Portal Components extraction with Package name 1`] = `
Array [
  Array [
    "https://example.com/images/logo",
  ],
  Array [
    "https://example.com/images/banner",
  ],
  Array [
    "media",
  ],
  Array [
    "media",
  ],
  Array [
    "media/logo.pdf",
    "",
  ],
  Array [
    "media/banner.pdf",
    "",
  ],
]
`;
